submissions/adeyemiJuwon/slimPHP/article/app/controllers/ArticleController.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace App\Controller;
+
+
+use DB\DB;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use PDO;
+use PDOException;
+
+
+
+class ArticleController{
+
+
+     private $db;
+    public function __construct(){
+        $this->db =(new DB())->connect();
+    }
+
+    public function index(Request $request, Response $response, $args){
```

Comment:
This isn't part of the endpoint you are ask to get

Reply:
Okay
Thank You @adaugochi